packages/utils/src/browser/detectVideoAlphaMode.ts

```diff
@@ -27,64 +27,58 @@ export async function detectVideoAlphaMode(): Promise<ALPHA_MODES>
-            0x30, 0x30, 0x3A, 0x30, 0x31, 0x2E, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x00,
-            0x00, 0x1F, 0x43, 0xB6, 0x75, 0xC7, 0xE7, 0x81, 0x00, 0xA0, 0xC2, 0xA1, 0xA0, 0x81, 0x00, 0x00,
-            0x00, 0x82, 0x49, 0x83, 0x42, 0x00, 0x00, 0x10, 0x00, 0x16, 0x00, 0x38, 0x24, 0x1C, 0x18, 0x4A,
-            0x00, 0x00, 0x20, 0x20, 0x00, 0x11, 0xBF, 0xFF, 0xFF, 0x8A, 0xFE, 0x00, 0x00, 0x75, 0xA1, 0x9D,
-            0xA6, 0x9B, 0xEE, 0x81, 0x01, 0xA5, 0x96, 0x82, 0x49, 0x83, 0x42, 0x00, 0x00, 0x10, 0x00, 0x16,
-            0x00, 0x38, 0x24, 0x1C, 0x18, 0x4A, 0x00, 0x00, 0x20, 0x20, 0x00, 0x48, 0x40, 0x1C, 0x53, 0xBB,
-            0x6B, 0x91, 0xBB, 0x8F, 0xB3, 0x81, 0x00, 0xB7, 0x8A, 0xF7, 0x81, 0x01, 0xF1, 0x82, 0x01, 0x71,
-            0xF0, 0x81, 0x03
-        ])], { type: 'video/webm' }));
-        video.load();
-
-        await new Promise<void>((resolve) =>
+        function loadVideo(src: string): Promise<HTMLVideoElement | null>
         {
-            function wait()
+            return new Promise<HTMLVideoElement | null>((resolve) =>
             {
-                if (video.readyState <= 1)
-                {
-                    setTimeout(wait, 1);
-                }
-                else
+                const video = document.createElement('video');
+
+                video.onerror = () => resolve(null);
+                video.autoplay = false;
+                video.crossOrigin = 'anonymous';
+                video.preload = 'auto';
+                video.src = src;
+                video.load();
+
+                function wait()
                 {
-                    resolve();
+                    if (video.readyState <= 1)
+                    {
+                        setTimeout(wait, 1);
+                    }
+                    else
+                    {
+                        resolve(video);
+                    }
                 }
+
+                wait();
+            });
+        }
+
+        let video: HTMLVideoElement | null = null;
+
+        for (const src of [
+            // WebM VP9
+            // eslint-disable-next-line max-len
+            'data:video/webm;base64,GkXfo59ChoEBQveBAULygQRC84EIQoKEd2VibUKHgQJChYECGFOAZwEAAAAAAAIOEU2bdLpNu4tTq4QVSalmU6yBoU27i1OrhBZUrmtTrIHWTbuMU6uEElTDZ1OsggEnTbuMU6uEHFO7a1OsggH47AEAAAAAAABZAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVSalmsCrXsYMPQkBNgIxMYXZmNjAuNC4xMDBXQYxMYXZmNjAuNC4xMDBEiYhARAAAAAAAABZUrmvMrgEAAAAAAABD14EBc8WIUzy3iLLKnmKcgQAitZyDdW5kiIEAhoVWX1ZQOYOBASPjg4QCYloA4JSwgQK6gQKagQJTwIEBVbCEVbmBARJUw2dAf3Nzn2PAgGfImUWjh0VOQ09ERVJEh4xMYXZmNjAuNC4xMDBzc9pjwItjxYhTPLeIssqeYmfIpEWjh0VOQ09ERVJEh5dMYXZjNjAuNi4xMDAgbGlidnB4LXZwOWfIokWjiERVUkFUSU9ORIeUMDA6MDA6MDAuMDQwMDAwMDAwAAAfQ7Z1x+eBAKDCoaCBAAAAgkmDQgAAEAAWADgkHBhKAAAgIAARv///iv4AAHWhnaab7oEBpZaCSYNCAAAQABYAOCQcGEoAACAgAEhAHFO7a5G7j7OBALeK94EB8YIBrPCBAw==',
```

Comment:
So we could use `canPlayType` for video/webm and only check for VP9

Reply:
Maybe the VP8 case is unnecessary. Probably extremly unlikely that VP9 isn't supported but VP8 is.